src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RequesterNameAndAddress.java

```diff
@@ -2,14 +2,16 @@
 
 import lombok.Builder;
 import lombok.Getter;
-import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
 
 /**
  * Example: NAD+PO+G3380314:900++SCOTT'
+ * <br>
+ * Example: NAD+PO+++NORTH DOWN GP'
```

Comment:
As before, a short description of the difference in the two formats would be nice.